lib/api/kuzzle.js

```diff
@@ -103,7 +102,13 @@ class Kuzzle extends EventEmitter {
     this.rootPath = path.resolve(path.join(__dirname, '..', '..'));
 
     this.services = new Services(this);
-    this.internalEngine = new InternalEngine(this);
+    //  @todo use internalStorage
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)